examples/feature_flag/main.tf

```diff
@@ -0,0 +1,52 @@
+provider "aws" {
+  region = local.region
+}
+
+locals {
+  region = "us-east-1"
+  name   = "appconfig-ex-${replace(basename(path.cwd), "_", "-")}"
+
+  tags = {
+    Name       = local.name
+    Example    = local.name
+    Repository = "https://github.com/terraform-aws-modules/terraform-aws-appconfig"
+  }
+}
+
+################################################################################
+# AppConfig
+################################################################################
+
+module "deactivated_appconfig" {
+  source = "../../"
+
+  name   = local.name
+  create = false
+}
+
+module "appconfig" {
```

Comment:
we don't need another directory for this. just show this implementation in the `appconfig_hosted` example with its outputs